### PR TITLE
Updates to error checking and fonts

### PR DESCRIPTION
**Pull request by sichaox**
Added comments for readability.

### CODE REVIEW

File: tetrahedron.js
Comment (ndattani) [inline tetrahedron.js:1]: This title of this comit is "Did the upload properly" which would suggest that you're uploading the same file but in a different way, however the file has indeed changed, so this commit description isn't completely accurate.

**Review by ndattani — CHANGES_REQUESTED** [review]
😢 It looks like we're on the road to having this problem: https://github.com/HPQC-LABS/HPQC/pull/81#issuecomment-1512866564 again.

**Comment by ndattani** [comment]
I'm not sure why this comment: https://github.com/HPQC-LABS/HPQC/pull/118/commits/f589ec434e0ae8d4728b2d2f44057b4d6a248ac0#r1295326140 is not appearing here.

**Comment by sichaox** [comment]
> 😢 It looks like we're on the road to having this problem: [#81 (comment)](https://github.com/HPQC-LABS/HPQC/pull/81#issuecomment-1512866564) again.

I have removed .DS_Store file in the latest commit.

File: tetrahedron.js
Comment (sichaox) [reply tetrahedron.js:1]: I have used git ammend to rename the commit.

**Pull request merged**